src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx

```diff
@@ -1,53 +1,89 @@
 import styled from "styled-components"
 
 import { useArtworkFilterContext } from "../ArtworkFilterContext"
+import {
+  FollowedArtistList,
+  fetchFollowedArtists,
+} from "./fetchFollowedArtists"
 import { OptionText } from "./OptionText"
 
 const ToggleLink = styled(Text)`
   cursor: pointer;
   text-decoration: underline;
 `
 
-export const ArtistsFilter: FC = () => {
-  const { aggregations, ...filterContext } = useArtworkFilterContext()
-  const [expanded, setExpanded] = useState(false)
+const INITIAL_ARTISTS_TO_SHOW = 6
+
+interface ArtistsFilterProps {
+  relayEnvironment?: any
+  fairID?: string
+  user?: User
+}
 
+export const ArtistsFilter: FC<ArtistsFilterProps> = ({
+  fairID,
+  relayEnvironment,
+  user,
+}) => {
+  const { aggregations, ...filterContext } = useArtworkFilterContext()
   const artists = aggregations.find(agg => agg.slice === "ARTIST")
 
+  const [expanded, setExpanded] = useState(
+    filterContext.currentlySelectedFilters().artistIDs.length > 0
+  )
+  const [followedArtists, setFollowedArtists] = useState<FollowedArtistList>([])
+  const followedArtistSlugs = followedArtists.map(({ slug }) => slug)
+
+  useEffect(() => {
+    if (relayEnvironment && user) {
+      fetchFollowedArtists({ relayEnvironment, fairID }).then(data => {
+        setFollowedArtists(data)
+      })
+    }
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
   if (!(artists && artists.counts)) {
     return null
   }
 
   const artistsSorted = sortBy(artists.counts, ["name"])
-  const initialArtistsGroup = artistsSorted.slice(0, 6)
+  const initialArtistsGroup = artistsSorted.slice(0, INITIAL_ARTISTS_TO_SHOW)
   const remainingArtistsGroup =
-    artistsSorted.length > 6
-      ? artistsSorted.slice(6 - artistsSorted.length)
+    artistsSorted.length > INITIAL_ARTISTS_TO_SHOW
+      ? artistsSorted.slice(INITIAL_ARTISTS_TO_SHOW - artistsSorted.length)
       : []
 
   const toggle = () => setExpanded(!expanded)
 
+  const isFollowedArtistCheckboxSelected =
+    !!user &&
+    filterContext.currentlySelectedFilters()["includeArtworksByFollowedArtists"]
   const followedArtistCheckboxProps = {
     onSelect: value =>
       filterContext.setFilter("includeArtworksByFollowedArtists", value),
-    selected: Boolean(
-      filterContext.currentlySelectedFilters()[
-        "includeArtworksByFollowedArtists"
-      ]
-    ),
+    selected: isFollowedArtistCheckboxSelected,
   }
-
   const followedArtistArtworkCount = filterContext?.counts?.followedArtists ?? 0
 
-  const toggleSelection = (selected, name) => {
+  const toggleArtistSelection = (selected, slug) => {
     let artistIDs = filterContext.currentlySelectedFilters().artistIDs.slice()
     if (selected) {
-      artistIDs.push(name)
+      artistIDs.push(slug)
     } else {
-      artistIDs = artistIDs.filter(item => item !== name)
+      // When an artist is de-selected, if it is a followed artist _and_ that filter
```

Comment:
This is the intended behavior, I think?! This is what I'm describing in the PR body basically.